pythonwhat/test_exercise.py

```diff
@@ -12,6 +14,8 @@
 from inspect import Parameter as param
 from pythonwhat.signatures import sig_from_params, sig_from_obj
 from pythonwhat.State import set_converter
+# spec v2 functions
+from pythonwhat.check_syntax import F, Ex
```

Comment:
It's not necessary, but might be useful for illustration, since it's similar in form to Ex(), so you could show people that these are the same.

```{python}
test1 = F().check_list_comp(0).check_body()
test2 = check_list_comp(0).check_body()
```

Since seeing the second case might seem more magical to people.